hera_sim/utils.py

```diff
@@ -678,6 +678,24 @@ def find_baseline_orientations(
     return antpair2angle
 
 
+def tanh_window(x, x_min=None, x_max=None, scale_low=1, scale_high=1):
+    if x_min is None and x_max is None:
```

Comment:
Seems like the warning message could include the statement that one should provide either `x_min` or `x_max`

Reply:
Good catch, I'll fix it. Thanks!